### PR TITLE
Changes for FR 7.0

### PR DESCRIPTION
**Pull request by marcofanti**

### CODE REVIEW

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java [hunk 1]
```diff
@@ -115,89 +106,114 @@ public Action process(TreeContext context) throws NodeProcessException {
...
+                .sessionId(UUID.randomUUID().toString())
+                .operatorFlags(com.behaviosec.isdk.config.Constants.FLAG_FINALIZE_SESSION)
+                .notes("FR-V" + BehavioSecPlugin.currentVersion) 
+                .build();
+
+        Response response = null;
+        try {
+            response = restClient.makeCall(callReport);
+        } catch (BehavioSecException e) {
+            e.printStackTrace();
```
Comment (FrankGasparovic) [inline src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java:201]: Use logger instead of print

**Review by FrankGasparovic — CHANGES_REQUESTED** [review]

**Comment by marcofanti** [comment]
Removed all System.out.println from main branch.

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecBooleanEvaluator.java [hunk 2]
```diff
@@ -17,30 +17,28 @@
 
 package com.behaviosec.tree.nodes;
 
-import com.behaviosec.tree.config.NoBehavioSecReportException;
+/*import com.behaviosec.isdk.config.NoBehavioSecReportException;
+import com.behaviosec.isdk.entities.Report;
```
[the remark this answers is not in the document]
Comment (marcofanti) [reply src/main/java/com/behaviosec/tree/nodes/BehavioSecBooleanEvaluator.java]: Removed

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecScoreEvaluator.java [hunk 3]
```diff
@@ -17,33 +17,29 @@
 
 package com.behaviosec.tree.nodes;
 
-import com.behaviosec.tree.config.NoBehavioSecReportException;
-import org.forgerock.json.JsonValue;
+/*
+import com.behaviosec.isdk.config.NoBehavioSecReportException;
```
[the remark this answers is not in the document]
Comment (marcofanti) [reply src/main/java/com/behaviosec/tree/nodes/BehavioSecScoreEvaluator.java]: Removed

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java [hunk 4]
```diff
@@ -115,89 +106,114 @@ public Action process(TreeContext context) throws NodeProcessException {
...
-            nameValuePairs.add(new BasicNameValuePair(Constants.USER_ID, username));
-            String timingData = context.sharedState.get(Constants.DATA_FIELD).asString();
-
-            if (timingData != null) {
-                nameValuePairs.add(new BasicNameValuePair(Constants.TIMING,
-                                                          timingData));
+        List<NameValuePair> nameValuePairs = new ArrayList<>(2);
+        String username = context.sharedState.get(Constants.USERNAME).asString();
+
+        System.out.println("sendRequest: " + nameValuePairs);
```
[the remark this answers is not in the document]
Comment (marcofanti) [reply src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java]: Removed / changed to log...

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecBooleanEvaluator.java [hunk 5]
```diff
@@ -132,22 +149,32 @@ public BehavioSecBooleanEvaluator(@Assisted Config config) throws NodeProcessExc
...
+            booleanEvaluatorEvaluator.config.setAllowReplay(config.allowReplay());
+            booleanEvaluatorEvaluator.config.setAllowInTraining(config.allowInTraining());
+            booleanEvaluatorEvaluator.config.setAllowRemoteAccess(config.allowRemoteAccess());
+            booleanEvaluatorEvaluator.config.setAllowTabAnomaly(config.allowTabAnomaly());
+            booleanEvaluatorEvaluator.config.setAllowNumpadAnomaly(config.allowNumpadAnomaly());
+            booleanEvaluatorEvaluator.config.setAllowDeviceChanged(config.allowDeviceChanged());
+            booleanEvaluatorEvaluator.config.setAllowIPChange(config.allowIpChange());
+            booleanEvaluatorEvaluator.config.setAllowCopyOrPaste(config.allowCopyPaste());
+            boolean evaluation = booleanEvaluatorEvaluator.evaluate(bhsReport);
+            System.out.println("evaluation BehavioSecBooleanEvaluator = " + evaluation);
```
[the remark this answers is not in the document]
Comment (marcofanti) [reply src/main/java/com/behaviosec/tree/nodes/BehavioSecBooleanEvaluator.java]: Removed

File: src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java (hunk 1 above)
Comment (marcofanti) [reply src/main/java/com/behaviosec/tree/nodes/BehavioSecAuthNode.java:199]: Changed to logger

**Comment by marcofanti** [comment]
Updated doc and merged with master

**Pull request merged**